src/app/nav-menu/nav-menu.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { NavMenuComponent } from './nav-menu.component';
+
+describe('NavMenuComponent', () => {
+  let component: NavMenuComponent;
+  let fixture: ComponentFixture<NavMenuComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [ NavMenuComponent ]
+    })
+    .compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(NavMenuComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
```

Comment:
I'd also like to see a couple more unit tests - perhaps testing that the correct divs are created, the title is correct, perhaps check if data is bound correctly, etc.

Reply:
I am still unsure how to execute proper unit tests for something such as this.

Reply:
Essentially you're just testing if everything built correctly. please see https://angular.io/guide/testing and https://semaphoreci.com/community/tutorials/testing-components-in-angular-2-with-jasmine